atlas/portal/api/portal.api.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+package portal.pb;
+option go_package = "atlas/portal/pkg/pb";
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+service Portal{
+  rpc GetVersion (google.protobuf.Empty) returns (VersionResponse) {
+    option (google.api.http) = {
+      get: "/version"
+    };
+  }
+
+  rpc GetInfo  (GetInfoRequest) returns (GetInfoResponse) {
+    option (google.api.http) = {
+      post: "/getInfo"
```

Comment:
Please stick to single word paths and differentiate actions by HTTP request methods. For example GET /info is GetInfo, POST /info is SetInfo.

Reply:
In addition to @amikhailau words: take a look [this article](https://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#restful), it has some useful examples